tests/python/unittest/test_tir_transform_lower_tvm_builtin.py

```diff
@@ -133,11 +133,45 @@ def check_packed_func(target="llvm"):
     tvm.ir.assert_structural_equal(alloca_shape, expected_stmt, map_free_vars=True)
 
 
-def test_packed_func():
+def test_lower_packed_func():
     check_packed_func("llvm")
```

Comment:
Can you add a check also for the C backend? I tried this in AOT, but the C backend appears to no handling return values correctly.

Reply:
i believe previously it was only works for non-C backend, but recent PR might added support

Reply:
Is the PR that adds non-C support already merged? If so, I think it should be ok to add a test. If not, we might add a TODO and add the test when the PR gets merged

Reply:
I agree a TODO is a good idea